Cargo.toml

```diff
@@ -40,19 +40,21 @@ console_error_panic_hook = "0.1.7"
 futures-util = "0.3.27"
 http = "0.2.6"
 js-sys = "0.3.49"
-matrix-sdk-common = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "bcbb7c61f87b3a35cb2df5e70c62a0e5b846b506", features = ["js"] }
-matrix-sdk-indexeddb = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "bcbb7c61f87b3a35cb2df5e70c62a0e5b846b506" }
-matrix-sdk-qrcode = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "bcbb7c61f87b3a35cb2df5e70c62a0e5b846b506", optional = true }
+matrix-sdk-common = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c2bb76029ae6d99c741727e0f87abcd734377016", features = ["js"] }
+matrix-sdk-indexeddb = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c2bb76029ae6d99c741727e0f87abcd734377016" }
+matrix-sdk-qrcode = { git = "https://github.com/matrix-org/matrix-rust-sdk", rev = "c2bb76029ae6d99c741727e0f87abcd734377016", optional = true }
 serde_json = "1.0.91"
 serde-wasm-bindgen = "0.5.0"
 tracing = { version = "0.1.36", default-features = false, features = ["std"] }
 tracing-subscriber = { version = "0.3.14", default-features = false, features = ["registry", "std"] }
 wasm-bindgen = "=0.2.84"
 wasm-bindgen-futures = "0.4.33"
 zeroize = "1.6.0"
+base64 = "0.13.0"
+rmp-serde = "1.1.2"
```

Comment:
do we particularly need these as direct dependencies now?

Reply:
I don't know what's the correct way to do it. If I don't add it the build is failing because it's not sure what version to use, but it's due to the Cargo.lock. 
I'll update the Cargo.lock instead.